SUI/SUI/Filters/CustomSliderTwoView.swift

```diff
@@ -0,0 +1,114 @@
+// CustomSliderTwoView.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Кастомный слайдер
+struct CustomSliderTwoView: View {
+    // MARK: - Private Constants
+
+    private enum Constants {
+        static let paddingNumber: CGFloat = 80
+        static let circleWidthNumber: CGFloat = 30
+        static let stepNumber = 500
+        static let dolarString = "$"
+        static let minValueLocationXNumber: CGFloat = 8
+    }
+
+    // MARK: - Public properties
+
+    @State var minValue: CGFloat
+    @State var maxValue: CGFloat
+    var totalWidth: CGFloat = UIScreen.main.bounds.width - Constants.paddingNumber
+
+    var body: some View {
+        ZStack(alignment: .leading) {
+            Rectangle()
+                .fill(.purple.opacity(0.2))
+                .frame(height: 10)
+            Rectangle()
+                .fill(.purple)
+                .frame(width: (maxValue - minValue) + Constants.circleWidthNumber, height: 10)
+                .offset(x: minValue + Constants.circleWidthNumber)
+            HStack {
+                makeMinValueCircleView
+                makeMaxValueCircleView
+            }
+        }
+        .offset(y: -80)
+        .padding()
+    }
+
+    // MARK: - Private properties
+
+    private var makeMaxValueCircleView: some View {
+        VStack {
+            Circle()
+                .fill(.orange)
+                .frame(width: 30, height: 30)
+                .background(
+                    Circle()
+                        .stroke(.white, lineWidth: 5)
+                )
+                .offset(x: maxValue)
+                .gesture(
+                    DragGesture()
+                        .onChanged { value in
+                            if
```

Comment:
```suggestion
                       if
                                 value.location.x <= totalWidth, value.location.x >= minValue,
                                 value.location.x > Constants.minValueLocationXNumber
                             {
                                 maxValue = value.location.x
                             }
```
В идеале это во вьюМодель выносить